app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -125,21 +125,44 @@
     }
   }
 
-  .shopfront_closed_message, .shopfront_hidden_message {
-    padding: 15px;
-    border-radius: 5px;
-  }
+  .closed-shop-header {
+    background-color: $grey-650;
+    color: $white;
 
-  .shopfront_closed_message {
-    border: 2px solid #eb4c46;
-  }
+    h4 {
+      color: $white;
+    }
 
-  .shopfront_closed_message {
-    margin: 2em 0em;
+    p {
+      line-height: 2.5rem;
```

Comment:
this is a quite large `line-height` for a paragraph are we sure it's not margin what we need here?

Reply:
ping @luisramos0

Reply:
yes, you are correct, I fixed it :+1: